### PR TITLE
remove old tooltip component and styles

### PR DESCRIPTION
**Pull request by brad-decker**
The old tooltip component was only used in two places. Removing those usages
was simple and straight forward. So, instead of colocating the old tooltip
styles with the deprecated tooltip component, I removed all old styles and
made tooltip-v2 now simply 'tooltip' and removed the deprecated component.

### CODE REVIEW

File: ui/app/css/index.scss [hunk 1]
```diff
@@ -22,5 +22,9 @@
 @import './itcss/objects/index';
 @import './itcss/components/index';
 @import './itcss/trumps/index';
-// Other imports
+
+/*
+  Third Party Library Styles
```
Comment (rekmarks) [inline ui/app/css/index.scss:27]: nit:
```suggestion
 * Third Party Library Styles
```

**Review by rekmarks — APPROVED** [review]
LGTM!

~Just one non-blocking nit.~

File: ui/app/css/index.scss (hunk 1 above)
Comment (rekmarks) [reply ui/app/css/index.scss:27]: There should be a lint rule for this.

Comment (rekmarks) [reply ui/app/css/index.scss:27]: There is a lint rule for this!

Comment (rekmarks) [reply ui/app/css/index.scss:27]: But it doesn't do what I want 😢

Comment (brad-decker) [reply ui/app/css/index.scss:27]: The comment style matches the other comment styles in this file and other scss files, at least some of which are pre-existing. I agree we should have an enforced comment style in scss files, though I'm curious if that has been decided and whether or not I can apply that to this file when I update it to remove all of the itcss imports?

Comment (brad-decker) [reply ui/app/css/index.scss:27]: gotta find a style-lint plugin for it ;)

Comment (rekmarks) [reply ui/app/css/index.scss:27]: TIL it's a good idea to consider the file extension when raving about styles 🤦‍♂️

**Pull request merged**